### PR TITLE
[Docs-1448] create product folders using new names

### PR DESCRIPTION
**Pull request by pattyhair**
Parking this PR until a final decision is made for product names.

### CODE REVIEW

File: cloudbees-build-test-automation/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# CloudBees Build and Test Automation Examples
```
Comment (tfoxnc) [inline cloudbees-build-test-automation/README.md:1]:
```suggestion
# CloudBees Build and Test Automation examples
```

File: cloudbees-cd/README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# CloudBees CD Examples
```
Comment (tfoxnc) [inline cloudbees-cd/README.md:1]:
```suggestion
# CloudBees CD examples
```

File: cloudbees-ci/README.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# CloudBees CI Examples
```
Comment (tfoxnc) [inline cloudbees-ci/README.md:1]:
```suggestion
# CloudBees CI examples
```

File: cloudbees-feature-flags/README.md [hunk 4]
```diff
@@ -0,0 +1,3 @@
+# CloudBees Feature Flags Examples
```
Comment (tfoxnc) [inline cloudbees-feature-flags/README.md:1]:
```suggestion
# CloudBees Feature Flags examples
```

File: cloudbees-rollout/README.md [hunk 5]
```diff
@@ -0,0 +1,3 @@
+# CloudBees Rollout Examples
```
Comment (tfoxnc) [inline cloudbees-rollout/README.md:1]: CloudBees Rollout is now CloudBees Feature Flags, so this directory can be deleted.

File: cloudbees-sdm/README.md [hunk 6]
```diff
@@ -0,0 +1,3 @@
+# CloudBees SDM Examples
```
Comment (tfoxnc) [inline cloudbees-sdm/README.md:1]:
```suggestion
# CloudBees SDM examples
```

File: flow-on-kubernetes/README.md [hunk 7]
```diff
@@ -1,12 +1,3 @@
-# CloudBees Flow Installation Examples
+# CloudBees CD Examples
```
Comment (tfoxnc) [inline flow-on-kubernetes/README.md:1]:
```suggestion
# Cloudbees CD Kubernetes examples
```

File: cloudbees-cd/README.md (hunk 2 above)
Comment (KimberlyFields) [inline cloudbees-cd/README.md:1]:
```suggestion
# CloudBees CD examples
```

File: cloudbees-cd/kubernetes/README.md [hunk 8]
```diff
@@ -0,0 +1,13 @@
+# CloudBees CD Examples
```
Comment (KimberlyFields) [inline cloudbees-cd/kubernetes/README.md:1]:
```suggestion
# CloudBees CD examples
```

File: cloudbees-ci/README.md (hunk 3 above)
Comment (KimberlyFields) [inline cloudbees-ci/README.md:1]:
```suggestion
# CloudBees CI examples
```

File: cloudbees-feature-flags/README.md (hunk 4 above)
Comment (KimberlyFields) [inline cloudbees-feature-flags/README.md:1]:
```suggestion
# CloudBees Feature Flags examples
```

File: cloudbees-rollout/README.md (hunk 5 above)
Comment (KimberlyFields) [inline cloudbees-rollout/README.md:1]:
```suggestion
# CloudBees Rollout examples
```

File: flow-on-kubernetes/README.md (hunk 7 above)
Comment (KimberlyFields) [inline flow-on-kubernetes/README.md:1]:
```suggestion
# CloudBees CD examples
```

File: cloudbees-sdm/README.md (hunk 6 above)
Comment (KimberlyFields) [inline cloudbees-sdm/README.md:1]:
```suggestion
# CloudBees SDM examples
```

File: cloudbees-sdm/README.md [hunk 9]
```diff
@@ -0,0 +1,3 @@
...
+
+Files in this folder complement mentions in CloudBees SDM product documentation, found [here](https://https://docs.cloudbees.com/docs/cloudbees-sdm/latest/). As such, they are not standalone.
```
Comment (KimberlyFields) [inline cloudbees-sdm/README.md:3]: This is not the format you used for the other products.

File: cloudbees-rollout/README.md (hunk 5 above)
Comment (KimberlyFields) [inline cloudbees-rollout/README.md:1]: Why is there a readme.md file for rollout (old product name)? You have a readme.md file for feature flags.

File: cloudbees-sdm/README.md (hunk 9 above)
Comment (pattyhair) [reply cloudbees-sdm/README.md:3]: Made requested changes.

File: cloudbees-rollout/README.md (hunk 5 above)
Comment (pattyhair) [reply cloudbees-rollout/README.md]: Done.

Comment (pattyhair) [reply cloudbees-rollout/README.md]: Removed

**Pull request merged**